src/windowsservercore/ltsc2019/webassembly/Dockerfile

```diff
@@ -81,3 +81,5 @@ RUN curl -SL --output vs_buildtools.exe https://aka.ms/vs/16/release/vs_buildtoo
     --add Microsoft.VisualStudio.Component.Windows10SDK.17763 \
 || IF "%ERRORLEVEL%"=="3010" EXIT 0) \
     && del /q vs_buildtools.exe
+
+RUN del /q /f /s %TEMP%\*
```

Comment:
Rather than having this in a separate RUN instruction, add this as part of the previous RUN instruction after deleting vs_buildtools.exe.  Doing that will save on overall image size.  By having a separate RUN instruction that deletes from TEMP it means the previous image layer kept the complete contents of TEMP and actually carries that with the image. So that's not a best practice.

Reply:
I wonder whether it would make sense to squash the image in the end?

Reply:
Ah, the squash option is experimental and requires docker daemon with experimental features enabled. So I guess it doesn't apply here.